app/models/rental.py

```diff
@@ -0,0 +1,18 @@
+from flask import current_app
+from app import db
+from datetime import datetime, timedelta
+
+class Rental(db.Model):
+    rental_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    customer_id = db.Column(db.Integer, db.ForeignKey('customer.customer_id'), primary_key=True)
+    video_id = db.Column(db.Integer, db.ForeignKey('video.video_id'), primary_key=True)
+    due_date = db.Column(db.DateTime, default=datetime.utcnow() - timedelta(days=7)) #set time to 7 days in future
+
```

Comment:
You may want to add a check in date to know if the user returned this rental:

```suggestion
    check_in_date = db.Column(db.DateTime, nullable=True)
```